src/reporting/Interfaces.sol

```diff
@@ -34,12 +33,22 @@ contract IMarket is Typed {
     function isDoneWithAllReporters() constant public returns (bool);
     function isDoneWithLimitedReporters() constant public returns (bool);
     function getReportingTokenOrZeroByPayoutDistributionHash(bytes32) constant public returns (ReportingToken);
+    function getFinalizationTime() constant public returns (uint);
```

Comment:
Style: Prefer using explicitly sized types rather than implicitly sized types.  In this case, `uint256`